ivre/utils.py

```diff
@@ -197,7 +197,8 @@ def ip2bin(ipval):
     if len(ipval) == 16:
         return ipval
     if len(ipval) == 4:
-        return b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff' + ipval
+        return b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff' + \
+            bytes(ipval, 'utf-8')
```

Comment:
Is this necessary? I **think** ipval should already be raw bytes here (or at least this is what we assume).